### PR TITLE
chips/earlgrey/chip_config: increase CW310 FPGA clock frequencies

### PR DESCRIPTION
**Pull request by lschuermann**
### Pull Request Overview

This is in accordance with an upstream change [1]. While the register set and other chip attributes are frozen for EarlGrey, the clock frequencies are not. This change will also be backported to the lowRISC/opentitan `earlgrey_es` branch [2].

[1]: https://github.com/lowRISC/opentitan/pull/19368
[2]: https://github.com/lowRISC/opentitan/pull/19479

### Testing Strategy

This pull request was tested by running Tock on latest OpenTitan master on a CW310 FPGA.

### TODO or Help Wanted

N/A

### Documentation Updated

- [x] ~Updated the relevant files in `/docs`,~ or no updates are required.

### Formatting

- [x] Ran `make prepush`.

### CODE REVIEW

**Comment by lschuermann** [comment]
Ping @cfrantz

**Comment by bradjc** [comment]
How does this relate to https://github.com/tock/tock/blob/0a2b74eba355dba42907fb6f5d693166ad4539be/boards/opentitan/earlgrey-cw310/Makefile#L18

**Comment by lschuermann** [comment]
> How does this relate to
> 
> https://github.com/tock/tock/blob/0a2b74eba355dba42907fb6f5d693166ad4539be/boards/opentitan/earlgrey-cw310/Makefile#L18

Yeah ... good point. We'll want to put this PR on hold until we have a new stable bitstream to increment to. My bad.

**Review by cfrantz — APPROVED** [review]
LGTM after https://github.com/lowRISC/opentitan/pull/19479 gets submitted on the `earlgrey_es` branch.

We'll need to update the OpenTitan SHA afterwards.

Note that the changes to clock frequencies don't change any of the Earlgrey hardware implementation - they only change the base clocks for the FPGA synthesis.

**Comment by bradjc** [comment]
@cfrantz Is there an updated bitstream?

**Comment by cfrantz** [comment]
> @cfrantz Is there an updated bitstream?

I've tested this change with opentitan/master@HEAD and with opentitan/earlgrey_es@7df72d61f536e9babb343219232783e5e49f7ceb.  Both worked.

The earlgrey_es bitstream is available from the opentitan-bitstreams GCP bucket here:  https://storage.googleapis.com/opentitan-bitstreams/earlgrey_es/bitstream-7df72d61f536e9babb343219232783e5e49f7ceb.tar.gz

I think this can go in as long as we update the `OPENTITAN_SUPPORTED_SHA`.

**Comment by bradjc** [comment]
Awesome!

**Comment by bradjc** [comment]
Replaced by #3703

**Pull request closed without merging**
